### PR TITLE
Add a new test.js file

### PR DESCRIPTION
**Pull request by SgbVGvQz-2**
Please accept PR

### CODE REVIEW

**Comment by SgbVGvQz-1** [comment]
Add more comments to code.

**Comment by SgbVGvQz-2** [comment]
How about now?

**Comment by SgbVGvQz-1** [comment]
Looks good.

**Comment by SgbVGvQz-2** [comment]
Request to reopen pull request.

**Comment by SgbVGvQz-1** [comment]
> 
> 
> Request to reopen pull request.

Ok

File: test.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Test message.
```
Comment (SgbVGvQz-2) [inline test.js]: Change this.

File: test.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+console.log('test');
```
Comment (SgbVGvQz-2) [inline test.js]: Seview started

File: test.js (hunk 1 above)
Comment (SgbVGvQz-1) [reply test.js]: done

**Pull request merged**